Source/Blazorise/wwwroot/dragDrop.js

```diff
@@ -20,6 +20,38 @@ export function destroy(element, elementId) {
     element.removeEventListener('dragstart', dragStartHandler);
 }
 
+export function InitializeThrottledDragEvents(element, elementId, dotnetAdapter)
+{
+    element = getRequiredElement(element, elementId);
+    if (!element)
+        return;
+
+    let timeOutForDrag = null;
+    let timeOutForDragOver = null;
+
+    function throttledDragHandler(e) {
+        e.preventDefault();
+        if (!timeOutForDrag) {
+            timeOutForDrag = setTimeout(function () {
```

Comment:
Can we also check if the mouse position is changed? Even if I hold the mouse while dragging the events are still propagated to .NET.

Reply:
I believe we should maintain the browser behaviour. The point is that it keeps being called while drag is happening.

The timeOut seems like the only necessary adjustment due to the nature interop.

Reply:
Ok